notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<p>Please explain why "A simple model like this corresponds exactly to a classic <a href="https://ieeexplore.ieee.org/abstract/document/4781121" target="_blank">matrix factorization</a> approach." or drop the sentence. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Did you explain why this approach corresponds to classic matrix factorization? </p>